extensions/copyq-clipboard-manager/package.json

```diff
@@ -5,18 +5,13 @@
   "description": "Lists CopyQ clipboard entries and allows user to paste them.",
   "icon": "command-icon.png",
   "author": "andrewcincotta",
-  "contributors": [
-    "pernielsentikaer"
-  ],
-  "categories": [
-    "Productivity"
-  ],
+  "categories": ["Productivity"],
   "license": "MIT",
   "commands": [
     {
-      "name": "view-clipboard-entries",
-      "title": "View Default Tab",
-      "description": "Lists CopyQ clipboard entries and allows user to select/paste them.",
+      "name": "clipboard-manager",
```

Comment:
I would keep the old name `view-clipboard-entries` since the people using it already has set up preferences and maybe an alias or hotkey for the command, this will be erased